index.html

```
@@ -721,8 +727,8 @@ <h3 class="font-headline font-extrabold text-lg sm:text-xl">2026 Timeline</h3>
     <div class="text-6xl mb-4">🔍</div>
     <h3 class="text-2xl font-bold text-zinc-900 dark:text-white mb-2">No organizations match your current filters.</h3>
     <p class="text-zinc-500 dark:text-zinc-400 mb-8">Try adjusting your search or clearing some filters.</p>
-    <button onclick="clearAllFilters()" class="px-8 py-3 bg-primary text-white rounded-xl font-bold hover:bg-orange-600 transition-colors shadow-lg shadow-primary/20">
-      Clear All Filters
+    <button id="clearFiltersBtn" class="px-8 py-3 bg-primary text-white rounded-xl font-bold hover:bg-orange-600 transition-colors shadow-lg shadow-primary/20">
```

HTMLHint (1.9.2)

[error] The id value [ clearFiltersBtn ] must be unique.

(id-unique)